File: Swift-C++-Testsuite/test/arithmetic-operators/Addition/Swift/SimpleAdd.swift
@main
struct MyMain {

    static func main() {
        // The code of this method can't be directly placed inside
        // the main function or it would be removed by the compiler
        // due to mandatory optimizations.
        var _ = simpleAdd(x: 1, y: 2)
    }

    static func simpleAdd(x: Int, y: Int) -> Int {
        var a = x
        var b = y
        var c = a + 41
        return c
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/SimpleAdd.cpp
 
#include <iostream> 

struct MyMain {

    static void main() {
        // The code of this method can't be directly placed inside
        // the main function or it would be removed by the compiler
        // due to mandatory optimizations.
        int _ = simpleAdd(1); 
    }

    static int simpleAdd(int x) {
        int a = x;
        int b = a+ 41;
        return b;  
    }
}; 

int main() {
    MyMain::main(); 
    return 0; 
}